src/main/java/com/minecrafttas/common/server/PacketHandlerRegistry.java

```diff
@@ -0,0 +1,61 @@
+package com.minecrafttas.common.server;
+
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import org.apache.commons.lang3.ArrayUtils;
+
+import com.minecrafttas.common.server.Client.Side;
+import com.minecrafttas.common.server.exception.PacketNotImplementedException;
+import com.minecrafttas.common.server.exception.WrongSideException;
+import com.minecrafttas.common.server.interfaces.ClientPacketHandler;
+import com.minecrafttas.common.server.interfaces.PacketHandlerBase;
+import com.minecrafttas.common.server.interfaces.PacketID;
+import com.minecrafttas.common.server.interfaces.ServerPacketHandler;
+
+public class PacketHandlerRegistry {
+	private static final List<PacketHandlerBase> REGISTRY = new ArrayList<>();
+
+	public static void register(PacketHandlerBase handler) {
+		if (!REGISTRY.contains(handler)) {
+			REGISTRY.add(handler);
+		} else {
+			System.out.println("Warning...");
+		}
+	}
+
+	public static void unregister(PacketHandlerBase handler) {
+		if (REGISTRY.contains(handler)) {
+			REGISTRY.remove(handler);
+		} else {
+			System.out.println("Warning...");
+		}
+	}
```

Comment:
best logging ever

Reply:
Forgot about that... The server was in it's on package and I didn't create a logger for it at the time... Now that I moved it to the common package, I can use the common logger